packages/flutter_test/lib/src/event_simulation.dart

```diff
@@ -97,10 +104,20 @@ class KeyEventSimulator {
     }
     return keyCode;
   }
+  static String _getWebKeyCode(LogicalKeyboardKey key) {
+    String keyCode;
+    for (final String code in kWebToLogicalKey.keys) {
+      if (key.keyId == kWebToLogicalKey[code].keyId) {
+        keyCode = code;
```

Comment:
just return code here and return null at line 115?

Reply:
This code is copied from _getKeyCode() function, this style is there. so I follow same style.

Anyway I think the style you suggest is better too. :)